### ISSUE TITLE
missing library 

### ISSUE THREAD

**Issue by johnjake** [brandy-kay/ThoughtHub#1]
Hi Kay, 

Thank you for this I am currently learning spring boot also and found your project, best recent example in spb I guess, but I encounter a problem in **package com.sun.xml.internal.ws.spi.db does not exist**. 

Did you import this library manually? 

regards 
John

**Comment by brandyodhiambo** [comment]
Hi

Thanks for reaching out did you create the database manually in your sql
server?

If not please try create just like have given the instructions from the
readme the tell me if you still encounter the error

Kind regards

**Comment by johnjake** [comment]
Hi, 

Yea I created manually the db in phpmyadmin, I just delete the line or imports were the error occurs and also the unit test lines as it yields error at my ends and its working already but it redirects me to `AuthEntryPointJwt` or http = 401 `"Sorry, You're not authorized to access this resource."`

Regard

**Comment by brandyodhiambo** [comment]
You are testing it from post man?

**Comment by johnjake** [comment]
Yeah I used postman to test but it say http error = 400 bad request 

end point: `http://localhost:8089/api/auth/signup`

body:  
{
	"firstName": "brandy",
	"lastName": "kay",
	"username": "brandy-kay",
	"password": "password",
	"email": "brandy.kay@gmail.com"
}

![image](https://github.com/brandy-kay/ThoughtHub/assets/4056983/9c0eaf84-6fc4-4cc2-900b-219dcc41488a)

**Comment by brandyodhiambo** [comment]
Good so login using the username and password the get the access token and
add bearer token on post man to any endpoint you do

**Comment by johnjake** [comment]
The signup wasn't success because the http error says = 400 or bad request
